src/main/java/duke/command/Delete.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.exception.WrongIndexException;
+import duke.task.Task;
+
+/**
+ * Class for the delete command.
+ */
+public class Delete implements Command {
+    private final int operand;
```

Comment:
I get that `operand` refers to the parameter that the user puts in, but will it be clearer if it is named `taskIndex` instead?